### ISSUE TITLE
Email address field is missing from PFP entries

### ISSUE THREAD

**Issue by geriweitzman** [TomOnTime/phpresourcedir-issues#3]
Hugs Tom,

None of the listed professionals' email addresses seem to have been copied over from the old PFP database to the new site... Any way to add that in?

Thanks,
Geri

**Comment by Sheeri** [comment]
Email isn't missing; it's just that there aren't emails for everyone. In the old system, there were 22 email addresses marked "public", and in the new one, there are 23 entries with emails.

See Chris Russel under: http://beta.polyfriendly.org/list.php?category=Therapists

**Comment by geriweitzman** [comment]
Thanks for taking a look, Sheeri. So you are saying that the majority of
people in the directory just never submitted an email address, even when
lots of those same people submitted websites?

Geri

**Comment by Sheeri** [comment]
yeah. We _do_ have email addresses for a lot of entries, but they're not
marked 'public' - I think they were contact email addresses, for us to
contact folks about their listing, but not to give away...

-Sheeri

On Wed, Jun 8, 2016 at 10:45 AM, geriweitzman notifications@github.com
wrote:

> Thanks for taking a look, Sheeri. So you are saying that the majority of
> people in the directory just never submitted an email address, even when
> lots of those same people submitted websites?
> 
> Geri
> 
> On Wednesday, June 8, 2016, Sheeri notifications@github.com wrote:
> 
> > Email isn't missing; it's just that there aren't emails for everyone. In
> > the old system, there were 22 email addresses marked "public", and in the
> > new one, there are 23 entries with emails.
> > 
> > See Chris Russel under:
> > http://beta.polyfriendly.org/list.php?category=Therapists
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> > https://github.com/TomOnTime/phpresourcedir-issues/issues/3#issuecomment-224602207
> > ,
> > or mute the thread
> > <
> > https://github.com/notifications/unsubscribe/AIgsuXFHKJDr6Dy6U3dh3WL0Yv5-iS2iks5qJs4wgaJpZM4IJNKg
> > 
> > .
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/TomOnTime/phpresourcedir-issues/issues/3#issuecomment-224612262,
> or mute the thread
> https://github.com/notifications/unsubscribe/ABr4aEp8O2YTk--Mh-QLHGIX4KOYJrRkks5qJtV2gaJpZM4IJNKg
> .

## 
- Sheeri K. Cabral

**Comment by geriweitzman** [comment]
Would it be possible for me to get a list of those email addresses, so that
I can contact people for updated info?

Thanks,
Geri

**Comment by Sheeri** [comment]
Oh! and those non-public emails are in the notes_admin field, so
theoretically they show up in an admin interface?

On Wed, Jun 8, 2016 at 11:03 AM, Sheeri K. Cabral awfief@gmail.com wrote:

> yeah. We _do_ have email addresses for a lot of entries, but they're not
> marked 'public' - I think they were contact email addresses, for us to
> contact folks about their listing, but not to give away...
> 
> -Sheeri
> 
> On Wed, Jun 8, 2016 at 10:45 AM, geriweitzman notifications@github.com
> wrote:
> 
> > Thanks for taking a look, Sheeri. So you are saying that the majority of
> > people in the directory just never submitted an email address, even when
> > lots of those same people submitted websites?
> > 
> > Geri
> > 
> > On Wednesday, June 8, 2016, Sheeri notifications@github.com wrote:
> > 
> > > Email isn't missing; it's just that there aren't emails for everyone. In
> > > the old system, there were 22 email addresses marked "public", and in
> > > the
> > > new one, there are 23 entries with emails.
> > > 
> > > See Chris Russel under:
> > > http://beta.polyfriendly.org/list.php?category=Therapists
> > > 
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <
> > > https://github.com/TomOnTime/phpresourcedir-issues/issues/3#issuecomment-224602207
> > > ,
> > > or mute the thread
> > > <
> > > https://github.com/notifications/unsubscribe/AIgsuXFHKJDr6Dy6U3dh3WL0Yv5-iS2iks5qJs4wgaJpZM4IJNKg
> > > 
> > > .
> > 
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > https://github.com/TomOnTime/phpresourcedir-issues/issues/3#issuecomment-224612262,
> > or mute the thread
> > https://github.com/notifications/unsubscribe/ABr4aEp8O2YTk--Mh-QLHGIX4KOYJrRkks5qJtV2gaJpZM4IJNKg
> > .
> 
> ## 
> - Sheeri K. Cabral

## 
- Sheeri K. Cabral
